package.json

```diff
@@ -13,7 +14,7 @@
     "check": "lerna run check",
     "ci": "npm run check && npm run test && npm run build",
     "clean": "lerna run clean",
-    "docs": "typedoc --options typedoc.json",
+    "docs": "typedoc --options typedoc.json && gh-pages -d docs",
```

Comment:
can you make `gh-pages -d docs` it's own command, so that we can generate docs without doing the gh-pages stuff?

Something like `docs: "gp-pages: gh-pages -d docs"` and then docs:deploy: "npm run docs && npm run gh-pages"